EDDI/MainWindow.xaml.cs

```diff
@@ -460,44 +456,70 @@ private void windowLoaded(object sender, RoutedEventArgs e)
             }
         }
 
-        // Handle changes to the eddi tab
-        private void homeSystemChanged(object sender, TextChangedEventArgs e)
+        // Handle changes to the editable home system combo box
+        private void HomeSystemText_TextChanged(object sender, TextChangedEventArgs e)
         {
             EDDIConfiguration eddiConfiguration = EDDIConfiguration.FromFile();
-            if (eddiConfiguration.HomeSystem != eddiHomeSystemText.Text)
+            if (eddiConfiguration.HomeSystem != homeSystemDropDown.Text)
             {
-                eddiConfiguration.HomeSystem = string.IsNullOrWhiteSpace(eddiHomeSystemText.Text) ? null : eddiHomeSystemText.Text.Trim();
-                eddiConfiguration = EDDI.Instance.updateHomeSystem(eddiConfiguration);
+                string systemName = homeSystemDropDown.Text?.ToLowerInvariant();
+                if (systemName.Length > 1)
+                {
+                    systemList = systemList.Where(s => s.StartsWith(systemName, StringComparison.InvariantCultureIgnoreCase)).ToList();
+                    if (systemList.Count < 5)
+                    {
+                        systemList = StarMapService.GetStarMapSystemsPartial(systemName)
```

Comment:
Recommended: `systemList = StarMapService.GetStarMapSystemsPartial(systemName, false, false)`
There's no need to request more information than we need, and all we really need here is names.